code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1207,6 +1207,11 @@
 
 	victim.do_jitter_animation(2)
 
+	if(ishuman(victim))
+		var/mob/living/carbon/human/H = victim
+		victim.emote_burstscream()
+		H.adjustCloneLoss(20)
```

Comment:
Why

```suggestion
		victim.adjustCloneLoss(20)
```

Reply:
Idk tbh, I originally had it as brain damage so I just used what worked. I'll let an actual maintainer commit this if it's better or not.

Reply:
1) remove the emote, the victim is dead, it's stupid
2) you cannot psydrain non human, so the ischeck is useless
3) cloneloss is a living proc, so casting to human is useless